src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/endpoints/BitbucketCloudEndpoint/config-detail.jelly

```diff
@@ -1,6 +1,20 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core"
          xmlns:f="/lib/form">
+  <f:optionalBlock title="${%Enable cache}" field="enableCache" inline="true">
```

Comment:
remove enable cache and use 0 as the disable cache?

Reply:
The numbers are used to configure the shared caches, but there are also two instance caches, which are disabled is this checkbox is not checked ...